pkg/controllers/managementuser/pspdelete/pspdelete.go

```diff
@@ -25,6 +29,18 @@ type handler struct {
 
 func Register(ctx context.Context, userContext *config.UserContext) {
 	starter := userContext.DeferredStart(ctx, func(ctx context.Context) error {
+		clusterName := userContext.ClusterName
+		logrus.Infof("Checking cluster [%s] compatibility before registering pspdelete controller.", clusterName)
+		clusterLister := userContext.Management.Management.Clusters("").Controller().Lister()
+		err := podsecuritypolicy.CheckClusterVersion(clusterName, clusterLister)
+		if err != nil {
+			if errors.Is(err, podsecuritypolicy.ErrClusterVersionIncompatible) {
+				logrus.Infof("%v - will not register pspdelete controller for cluster [%s].", err, clusterName)
```

Comment:
During normal execution, I don't think it is helpful for the user to know that the PSP controller was not registered, which makes me lean towards this being a debug message.
```suggestion
				logrus.Debugf("%v - will not register pspdelete controller for cluster [%s].", err, clusterName)
```
I don't want to spam comments but this applies to the other version checks as well.